ocaml/default.nix

```diff
@@ -23,6 +23,7 @@
 , cairo
 , gtk2
 , zlib-oc
+, callPackage
```

Comment:
Should come from osuper?

Reply:
no, it doesn't even exist.. It comes from `pkgs.callPackage`.

Reply:
It exists and scopes you with the correct OCaml version

Reply:
```
          468|   dune_3 = lib.trace "${builtins.toJSON (osuper.callPackage)}" osuper.dune_3.overrideAttrs (o: {
```